flask_unchained/bundles/oauth/config.py

```diff
@@ -0,0 +1,27 @@
+from flask_unchained import BundleConfig
+
+
+class Config(BundleConfig):
+    """
+    Default configuration settings for the OAuth Bundle.
+    """
+
+    pass
+
+
+class TestConfig(Config):
+    """
+    Default test settings for the OAuth Bundle.
+    """
+
+    OAUTH_REMOTE_APP_GITHUB = dict(
+        # these are the kwargs that get passed to `OAuth.register_app`
+        consumer_key='a11a1bda412d928fb39a',
+        consumer_secret='92b7cf30bc42c49d589a10372c3f9ff3bb310037',
```

Comment:
I don't think you want to be committing this :)

(and it's probably a good idea to reset/refresh the secret from your GitHub account security panel, there are bots that scrape github for such secrets)

Reply:
It's from the oauthlib example here:
https://github.com/lepture/flask-oauthlib/blob/master/example/github.py

Reply:
oh interesting, ok, glad it's not *your* secret key